### PR TITLE
tell cobra the name flag is mandatory for set and delete, but not get

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Review by KirbyKatcher — APPROVED** [review]

File: pkg/cmd/properties.go [hunk 1]
```diff
@@ -31,14 +31,33 @@ func init() {
...
+
+	cmd.MarkPersistentFlagRequired(flagName)
+}
+
+// Some sub-commands need a name field to be mandatory, some don't.
+func addNameProperty(cmd *cobra.Command, isMandatory bool) {
+	flagName := "name"
+	var description string
+	if isMandatory {
+		description = "A mandatory field indicatig the name of a property in the namespace."
```
Comment (eamansour) [inline pkg/cmd/properties.go:53]: Typo - "indicatig"

Comment (techcobweb) [reply pkg/cmd/properties.go]: Fixed. Well spotted @eamansour !

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**